src/App.tsx

```diff
@@ -1,24 +1,251 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useState,
+  useEffect,
+  useCallback,
+  useMemo,
+} from 'react';
+import { v4 as uuidv4 } from 'uuid';
 import { UserWarning } from './UserWarning';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoNotification } from './components/TodoNotification';
+import {
+  getTodos,
+  createTodo,
+  updateTodo,
+  deleteTodo,
+} from './api/todos';
+import { TodoStatus } from './types/TodoStatus';
+import { Todo } from './types/Todo';
+import { TodoListContext } from './context/TodoListContext';
 
-const USER_ID = 0;
+const USER_ID = 10509;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(TodoStatus.All);
+  const [hasError, setHasError] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [completedTodoIdList, setCompletedTodoIdList] = useState<number[]>([]);
+
+  const showErrorMessage = (message: string) => {
+    setHasError(true);
+    setErrorMessage(message);
+  };
+
+  const handleCloseButton = useCallback(() => {
+    setHasError(!hasError);
+  }, [hasError]);
+
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setHasError(false);
+    }, 3000);
+
+    return () => {
+      clearTimeout(timeoutId);
+    };
+  }, [showErrorMessage]);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(todoList => setTodos(todoList))
+      .catch(() => showErrorMessage('Unable to load todos'));
+  }, [USER_ID]);
+
+  const visibleTodos = useMemo(() => {
+    let visibleTodoArray = todos;
```

Comment:
```suggestion
    return todos.filter 
```
    and inside filter add condition for active and completed